platform-hub-api/app/models/docker_repo.rb

```diff
@@ -0,0 +1,51 @@
+class DockerRepo < ApplicationRecord
+
+  NAME_REGEX = /\A[a-z]+[a-z0-9\-_\/]*\z/
+
+  include Audited
+
+  audited descriptor_field: :name, associated_field: :service
+
+  attr_readonly :name, :service_id
+
+  before_validation :add_project_shortname_to_name
+
+  belongs_to :service, -> { readonly }
+
+  scope :by_services, ->(ids) { where(service_id: ids) }
+
+  enum status: {
+    pending: 'pending',
+    ready: 'ready',
+    deleting: 'deleting',
```

Comment:
Just a thought. What if repo couldn't get created upstream for whatever reason? Should we add `failed` status (or similar) or we just keep it permanently in `pending` in such case?

Reply:
Cool 👍